runtime/integration-tests/src/xcm_mock/xcm_tests.rs

```diff
@@ -1319,15 +1318,13 @@ fn receive_relay_asset_with_trader_on_parachain() {
     });
 
     ParaA::execute_with(|| {
-        // ALICE gets amount - fee
-        assert_eq!(
-            parachain::Assets::balance(relay_asset_id_on_a, &ALICE),
-            amount - fee
-        );
+        let alice_amount = parachain::Assets::balance(relay_asset_id_on_a, &ALICE);
+        // ALICE gets number greater than 0 but less then amount
+        assert!(0 < alice_amount && alice_amount < amount - fee);
         // Fee sink gets fee
         assert_eq!(
             parachain::Assets::balance(relay_asset_id_on_a, XcmFeesAccount::get()),
-            fee
+            amount - alice_amount
         );
     });
 }
```

Comment:
the fee amount changed here, seems reasonable but xcm tx could fail for small amounts that succeeded before